packages/spotlight/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact spotlight module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `spotlight` navigation from the focused element clipped by an overflow container
```

Comment:
How about
```suggestion
- `spotlight` to find a navigable element when the focused element clipped by an overflow container
```
Or, I think it would be okay to leave it as is.